XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -2012,5 +2014,20 @@ private void ToggleExpandedState(Func<PDSection, bool> expand)
                 panel2.ResumeLayout();
             }
         }
+
+        private void SetupDeprecationBanner()
+        {
+            Banner.Visible = false;
+            if (Helpers.PostStockholm(xenObject.Connection))
+                return;
+
+            if ((int)API_Version.LATEST >= (int)API_Version.API_2_16)
+            {
+                Banner.BannerType = DeprecationBanner.Type.Warning;
+                Banner.WarningMessage = Messages.WARNING_PRE_CLOUD_VERSION_CONNECTION;
+                Banner.LinkUri = new Uri(InvisibleMessages.OUT_OF_DATE_WEBSITE);

```

Comment:
I think we need to set the LinkText as well, because when I saw Details I thought it might launch a pop-up with more information. Maybe use Messages.PATCHING_WIZARD_WEBPAGE_CELL instead?

Reply:
Good suggestion. The deprecation banner in ballooning page could change to use "Go to web page" link text as well.